agent/xds/rbac_test.go

```diff
@@ -340,17 +362,17 @@ func TestRemoveIntentionPrecedence(t *testing.T) {
 				},
 			},
 		},
-		"default-deny-allow-all-and-path-deny": {
```

Comment:
What happened to the prior case? Can you restore it?

Reply:
oh good catch

Reply:
This was just a rename blunder